src/index.ts

```diff
@@ -117,4 +128,70 @@ export class OpusPool {
     async getRewardsHistory(params: Parameters<typeof rewardsHistory>[1]): Promise<Array<RewardsDataPoint>> {
         return rewardsHistory(this, params);
     }
+
+    /**
+     * Generates mint transaction to mint osTokens from chosen Vault.
+     *
+     * @param params - params for request
+     * @param params.shares - Amount of osTokens to mint
+     * @param params.vault - A vault address
+     * @param params.referrer - Address of the referrer. Optional.
+     * @returns `MintTransactionData` for transaction to sign and broadcast
+     */
+    async buildMintTransaction(params: Parameters<typeof mintOsToken>[1]): Promise<StakingTransactionData> {
+        return mintOsToken(this, params);
+    }
+
+    /**
+     * Retrieves maximum amount of osTokens that can be minted by the user
+     *
+     * @param vault_address - A vault address
+     * @returns Amount of osTokens that can be minted
+     */
+    async getMaxMintForVault(vault_address: Hex): Promise<bigint> {
```

Comment:
we use `vault` as the name for vault addresses everywehere, I guess it should be here too. Also just generally using snake_case in js for vars is a lint issue